i18n/README-RU.md

```diff
@@ -121,7 +121,7 @@ __ВНИМАНИЕ__: Мы работаем над доведением Flame д
 
 ## Благодарности
 
- * [Fireslime](https://fireslime.xyz), команда ответственная за поддержку Flame.
+ * [Blue Fire](https://patreon.com/bluefireoss), команда ответственная за поддержку Flame.
```

Comment:
Also line 79 in this file (unfortunately, can't add a comment there)

Reply:
Done